src/go/rpk/pkg/adminapi/api_features.go

```diff
@@ -54,7 +54,7 @@ type LicenseProperties struct {
 // GetFeatures returns information about the available features.
 func (a *AdminAPI) GetFeatures(ctx context.Context) (FeaturesResponse, error) {
 	var features FeaturesResponse
-	return features, a.sendAny(
+	return features, a.sendToLeader(
```

Comment:
Was this wrong originally? same comment below

Reply:
Not wrong, but Core team suggested querying the leader in these 2 endpoints. They are GET requests and retrieving the information can be done from any node since they are replicated. It's just safer in the case one node is not up-to-date.